sdk/communication/communication-common/src/identifierModelSerializer.ts

```diff
@@ -119,23 +113,26 @@ export const serializeCommunicationIdentifier = (
   const identifierKind = getIdentifierKind(identifier);
   switch (identifierKind.kind) {
     case "communicationUser":
-      return { communicationUser: { id: identifierKind.communicationUserId } };
+      return {
```

Comment:
This is changing the default translation to wire format for `communicationUser` identifier. Intended? If yes, I think this deserves a CHANGELOG entry: `Always include 'rawId' when serializing identifiers to wire format` or some such.